File: simsu/1주차/120903_배열의_유사도.js
function solution(s1, s2) {
    let answer = 0;
    let wq = false;
    s1.map(x => {
        s2.map(y => {
            if(x==y && !wq) {
                wq = true;
                answer++;
            }
        })
        wq=false;
        }
    );
    return answer;
}

File: rolemadelen/programmers/120903_배열의_유사도.cpp
#include <string>
#include <vector>
#include <map>

using namespace std;

int solution(vector<string> s1, vector<string> s2) {
    map<string, int> mymap;
    
    for(string &s : s1) {
        mymap[s] += 1;
    }
    
    for(string &s : s2) {
        mymap[s] += 1;
    }
    
    auto it = mymap.begin();
    
    int cnt =0;
    while(it != mymap.end()) {
        if(it->second > 1) ++cnt;
        ++it;
    }
    
    return cnt;
}